models/SMSTenDLCBrand.ts

```diff
@@ -55,11 +55,11 @@ export enum SMSTenDLCBrandStockExchangeEnum {
 }
 export enum SMSTenDLCBrandBrandRelationshipEnum {
 
-    BASICACCOUNT = 'BASIC_ACCOUNT',
-    SMALLACCOUNT = 'SMALL_ACCOUNT',
-    MEDIUMACCOUNT = 'MEDIUM_ACCOUNT',
-    LARGEACCOUNT = 'LARGE_ACCOUNT',
-    KEYACCOUNT = 'KEY_ACCOUNT'
+    BASIC_ACCOUNT = 'BASIC_ACCOUNT',
```

Comment:
are these enum changes breaking changes?

Reply:
yep reverting changes back to normal